dronegen/tag.go

```diff
@@ -175,7 +176,11 @@ func tagPipelines() []pipeline {
 
 			// RPM/DEB package builds
 			for _, packageType := range []string{rpmPackage, debPackage} {
-				ps = append(ps, tagPackagePipeline(packageType, buildType{os: "linux", arch: arch, fips: fips}))
+				bt := buildType{os: "linux", arch: arch, fips: fips}
+				if packageType == "rpm" && arch == "amd64" {
```

Comment:
Could you add a comment here explaining why we're doing this. Similar to the one in build-package.sh.

Reply:
Should I add one here or at [#R291](https://github.com/gravitational/teleport/pull/11008/files#diff-2f00d76ba459d5bc8e89a1111b0d9a602c8d878bf63bb685c83be8ba801ea51bR291-R295) ? This simply sets the field value (which could be used anywhere) while R291-R295 does something with it.

Reply:
I think here is appropriate since we want to explain _why_ we're setting the centos7 flag when building x8664 rpms.

Reply:
Come to think of it I don't know that we should be checking the architecture here. The original fix only covered amd64, but I don't see any reason why the issue _wouldn't_ affect i386 and ARM as well. That being said, I don't have an easy way to test this. Do you have any thoughts here?

Reply:
Good question. I imagine those are less commonly used than amd64 but we'd probably fix them too. Let's do amd64 first so we can fix the most common use case and then follow up with ARM/32-bit (don't know if anyone's actually using it TBH). Can you use AWS account to spin up proper CentOS 7 boxes for testing? They should have ARM boxes.

Reply:
Also, looking at our downloads page, it doesn't look like we actually provide CentOS 7 compatible ARM binaries at all currently.

Reply:
I can test 64 bit ARM on AWS pretty easily, but as far as I am aware there are not any 32 bit CentOS or RHEL AMIs available on AWS. This makes it non-trivial to test for 32 bit issues. That being said, if the issue persists with 64 bit ARM it probably affects i386 and ARMv7 as well.